logbus/bus.go

```diff
@@ -132,11 +132,9 @@ func (b *Bus) WriteDeltaManifest(dm *logstream.DeltaManifest) {
 			DeltaManifest: dm,
 		},
 	}
-
 	b.rawMu.Lock()
+	defer b.rawMu.Unlock()
```

Comment:
is this intentionally -- didn't you make this change fairly recently? Did it turn out to introduce a race condition?

Reply:
Yes, it led to a race in certain circumstances. If the subscribers are added/removed frequently for instance. It ended up being less of a concern now that the `Write` operations don't block to the same degree.